src/app/(proper_react)/redesign/(authenticated)/user/dashboard/getUserDashboardState.tsx

```diff
@@ -507,7 +507,16 @@ export const getUserDashboardState = (
     return "UsUserScanInProgressResolvedBreaches";
   }
 
-  captureMessage(`InvalidUserState: ${JSON.stringify(contentProps)}`);
+  /* c8 ignore next 9 */
+  if (typeof window === "undefined") {
```

Comment:
Possibly the `server-only` package is of interest here: https://nextjs.org/docs/app/building-your-application/rendering/composition-patterns#keeping-server-only-code-out-of-the-client-environment

Reply:
Yes that is thanks! I'll file a follow to make `logging.ts` work in both contexts, we can use that here. I'd like to make this easier for next time.